settings.py

```diff
@@ -1,6 +1,6 @@
 import logging
 
-logging.basicConfig(level=logging.INFO, format='%(levelname)s: %(message)s')
+#logging.basicConfig(level=logging.INFO, format='%(levelname)s: %(message)s')
 logger = logging.getLogger(__name__)
 
```

Comment:
can you start to incorporate/merge your logging config into this one? Eventually, this will be the only logging config

Reply:
format="%(asctime)s [%(levelname)-5.5s]  %(message)s", as well as the handlers